src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Authentication failed: "
+                    + "Invalid username or password.", e);
```

Comment:
```suggestion
            logger.error("Authentication failed: ", e);
```
Will be ok, because we already have the message "Username or password are incorrect" in AuthenticationServiceImpl.class.